src/Lumi/Components2/Image.purs

```diff
@@ -187,6 +194,20 @@ resize props =
 round :: ImageModifier Thumbnail
 round = style_ mkRound
 
+cover :: PropsModifier ThumbnailProps
+cover =
+  propsModifier
+    _
+      { imgStyle = E.css { objectFit: E.str "cover" }
+      }
+
+contain :: PropsModifier ThumbnailProps
+contain =
+  propsModifier
+    _
+      { imgStyle = E.css { objectFit: E.str "contain" }
+      }
```

Comment:
I believe we should change these combinators to append to `imgStyle` instead of replacing its value, e.g.:
```purescript
cover :: PropsModifier ThumbnailProps
cover =
  propsModifier  \props -> props
    { imgStyle = props.imgStyle <> E.css { objectFit: E.str "cover" }
    }
```